test/source/tests/gmail.ts

```diff
@@ -126,6 +126,14 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       expect(urls.length).to.eq(1);
     }));
 
+    ava.default('mail.google.com - send password protected message', testWithBrowser('ci.tests.gmail', async (t, browser) => {
```

Comment:
The fix looks good, but why do we need a live test?
Is this because we don't check authorization in our '/api/message/upload'  handler in `backend-endpoints.ts`  (Flowcrypt API emulation)? We can try to add the extra checks there too, have you tried that? This way, I suppose, the previous compose test would suffice to cover the issue.

Reply:
You are right. 👍 
We indeed don't need live test for this.